database/postgres.js

```diff
@@ -0,0 +1,260 @@
+            postgres.end();
+            db = new Sequelize(dbName, config.username, config.password, {
+              dialect: 'postgres',
+              host: 'localhost',
+              logging: false
+            });
+            createModels();
+          });
+        }
+
+      });
+
+    let createModels =  function () {
+      console.log('creating Models');
+      featuresTable = db.define('features', {
+        house_id: {
+          type: Sequelize.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+          allowNull: false
+        },
+        type: {
+          type: Sequelize.STRING
+        },
+        year_built: {
+          type: Sequelize.INTEGER
+        },
+        heating: {
+          type: Sequelize.STRING
+        },
+        cooling: {
+          type: Sequelize.STRING
+        },
+        parking: {
+          type: Sequelize.STRING
+        },
+        lot: {
+          type: Sequelize.INTEGER
+        },
+        days_on_zillow: {
+          type: Sequelize.INTEGER
+        },
+        bedrooms: {
+          type: Sequelize.FLOAT
+        },
+        bathrooms: {
+          type: Sequelize.FLOAT
+        },
+        interiorheating: {
+          type: Sequelize.STRING
+        },
+        interiorcooling: {
+          type: Sequelize.STRING
+        },
+        appliances: {
+          type: Sequelize.STRING
+        },
+        kitchen: {
+          type: Sequelize.STRING
+        },
+        flooring: {
+          type: Sequelize.STRING
+        },
+        sqft: {
+          type: Sequelize.INTEGER
+        }
+      }, {timestamps: false});
+
+      cb();
+
+      }
+
+  }
+
+
+  var createParams = (numRecords) => {
+    let params = [];
+    features.createFeatures(numRecords).forEach(house => {
+      params.push({type:house.type,
+        year_built: house.year_built,
+        heating: house.heating,
+        cooling: house.cooling,
+        parking: house.parking,
+        lot: house.lot,
+        days_on_zillow: house.days_on_zillow,
+        bedrooms: house.bedrooms,
+        bathrooms: house.bathrooms,
+        interiorheating: house.int_heating,
+        interiorcooling: house.int_cooling,
+        appliances: house.appliances,
+        kitchen: house.kitchen,
+        flooring: house.flooring,
+        sqft: house.sqft});
+    });
+
+    return params;
+  }
+
+
+  //function to load feature mock data
+  var loadFeatures = (callback, numRecords = 100, params = []) => {
+    // fs.writeFileSync('/Users/apple/Code/RPT11/zillowclone/database/mockfeature.json', JSON.stringify(features.createFeatures()));
+
+    if (params.length === 0) {
+      features.createFeatures(numRecords).forEach(house => {
+        params.push([house.type, house.year_built, house.heating, house.cooling, house.parking, house.lot, house.days_on_zillow, house.bedrooms, house.bathrooms, house.int_heating, house.int_cooling, house.appliances, house.kitchen, house.flooring, house.sqft]);
+      });

```

Comment:
Why not use the createParams function above?

Reply:
Totally should do that.  I created the createParams function to have staticData for the 10M records this was basically the original code from FEC which only runs when 100 records is invoked.